vue-frontend/src/components/services/CTASection.vue

```diff
@@ -0,0 +1,153 @@
+            >Talk to our experts
+          </span>
+        </router-link>
+      </div>
+      <div
+        class="tw-hidden lg:tw-block tw-relative tw-w-[53%] xl:tw-w-[40%] 2xl:tw-w-[50%]"
+      >
+        <aspect-ratio height="50%">
+          <img
+            :src="ctaImages[0]"
+            :srcset="`${ctaImages[0]} 400w, ${ctaImages[1]} 800w`"
+            loading="lazy"
+            class="tw-absolute tw-right-0 tw-w-[100%] xl:tw-w-[95%] 2xl:tw-w-[66%] tw-h-full tw-object-cover tw-z-[2]"
+            alt="cta-image"
+          />
+        </aspect-ratio>
+        <div
+          class="swiper-content tw-mt-[3rem] xl:tw-mt-[6rem] lg:tw-w-[43%] xl:tw-w-[45%] 2xl:tw-w-[60%] tw-pl-4 md:tw-pl-0"
+        >
+          <swiper
+            :slidesPerView="1"
+            :centeredSlides="false"
+            :speed="5000"
+            :autoplay="{
+              delay: 3000,
+              disableOnInteraction: false,
+            }"
+            :loop="true"
+            :loopedSlides="50"
+            :spaceBetween="20"
+            :navigation="true"
+            @swiper="onSwiper"
+            class="swiper-container"
+          >
+            <swiper-slide
+              v-for="(slide, index) in slides"
+              :key="index"
+              @mouseover="playSwiper(false)"
+              @mouseleave="playSwiper(true)"
+              @touchstart.passive="playSwiper(false)"
+              @touchmove.passive="playSwiper(true)"
+              @touchend="playSwiper(true)"
+              class="tw-cursor-pointer"
+            >
+              <aspect-ratio
+                height="50%"
+                class="tw-mb-[80px] tw-border-solid tw-border-transparent tw-border-[1px]"
+              >
+                <img
+                  :src="slide.image[0]"
+                  class="tw-mb-10 tw-mr-0 tw-w-full tw-h-full tw-object-cover"
+                  alt="cta-slider-image"
+                />
+              </aspect-ratio>
+            </swiper-slide>
+          </swiper>
+        </div>
+      </div>
+    </div>
+  </section>
+</template>
+<script>
+import SwiperCore, { Pagination, Navigation, Autoplay } from "swiper";
+import { Swiper, SwiperSlide } from "swiper/vue";
+import ctabg400w from "@/assets/images/services/cta/cta-bg-400w.webp";
+import ctabg800w from "@/assets/images/services/cta/cta-bg-800w.webp";
+import cta400w from "@/assets/images/services/cta/cta-400w.webp";
+import cta800w from "@/assets/images/services/cta/cta-800w.webp";
+import slide1400w from "@/assets/images/services/cta/slide-1-400w.webp";
+import slide2400w from "@/assets/images/services/cta/slide-2-400w.webp";
+import AspectRatio from "../utils/AspectRatio.vue";
+
+SwiperCore.use([Pagination, Navigation, Autoplay]);
+export default {
+  components: { AspectRatio },
+  data() {
+    return {
+      contactURL: "/contact",
```

Comment:
Use it directly if it's used only in one place. We can create variable if its used at many places